src/MeshKernelNET/Api/IMeshKernelApi.cs

```diff
@@ -890,6 +890,14 @@ public interface IMeshKernelApi : IDisposable
         int Mesh2dDeleteEdge(int meshKernelId, double xCoordinate, double yCoordinate, double xLowerLeftBoundingBox,
                              double yLowerLeftBoundingBox, double xUpperRightBoundingBox, double yUpperRightBoundingBox);
 
+        /// <summary>
+        /// Deletes a mesh 2d edge given the index of the edge
+        /// </summary>
+        /// <param name="meshKernelId">Id of the grid state</param>
+        /// <param name="edgeIndex">The index of the edge to delete</param>
+        /// <returns> true if the edge has been deleted, false if not (the edge is outside the search radius) </returns>

```

Comment:
there is no search radius involved in this function, is there?

Reply:
No. Copy/paste error. Will fix.